homeassistant/components/matrix/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "matrix",
   "name": "Matrix",
   "documentation": "https://www.home-assistant.io/integrations/matrix",
-  "requirements": ["matrix-client==0.4.0"],
+  "requirements": ["matrix-client==0.4.0", "markdown==3.3.6"],
```

Comment:
It feels wrong to add `markdown` as a requirement to this integration to do this conversion.

Reply:
Could you clarify?

Most popular chat clients ([Slack](https://slack.com/help/articles/202288908-Format-your-messages), [Teams](https://support.microsoft.com/en-us/office/use-markdown-formatting-in-teams-4d10bd65-55e2-4b2d-a1f3-2bebdcd2c772), etc) provide some sort of markdown support, including the "official" Element.io client (though their documentation doesn't include a section on message formatting, the basic markdown syntax works).

Is there a simpler alternative? I felt it was better to leverage a maintained package rather than trying to roll a one off implementation of the markdown standard.

Reply:
Home Assistant exposes features for chat clients as-is. We shouldn't try to add extra features like markdown support for Matrix. If Matrix would support Markdown, it would be ok. Here we fake the feature, that's not ok.

Reply:
Makes sense, I will remove the markdown support.

Reply:
@balloob Done.